src/PerformanceTests/Tests/Categories/SerializerFixture.cs

```diff
@@ -6,7 +6,7 @@
 namespace Categories
 {
 
-    [TestFixture(Description = "Serializer differences", Category = "Performance"), Explicit]
+    [TestFixture(Description = "Serializer differences", Category = "Performance")]
```

Comment:
If we want to run this for dotnet core then we must select a supported transport like RabbitMQ but because that is running remotely it is very hard to measure any difference in performance because serialization isn't the biggest issue.